indy_node/test/context/test_send_get_context.py

```diff
@@ -0,0 +1,310 @@
+import json
+
+import pytest
+
+from indy_common.constants import GET_CONTEXT, CONTEXT_TYPE, RS_TYPE, CONTEXT_NAME, CONTEXT_VERSION
+from indy_node.test.context.helper import W3C_BASE_CONTEXT
+from plenum.common.exceptions import RequestNackedException
+
+from plenum.common.constants import DATA, NAME, VERSION, TXN_METADATA, TXN_METADATA_SEQ_NO
+
+from plenum.common.types import OPERATION
+
+from plenum.test.helper import sdk_sign_and_submit_req, sdk_get_and_check_replies
+
+from indy_node.test.api.helper import sdk_write_context
+from indy_node.test.helper import createUuidIdentifier, modify_field
+
+
+TEST_CONTEXT_NAME = "Base_Context"
+TEST_CONTEXT_VERSION = "1.0"
+
+
```

Comment:
Please add tests to `indy_node/test/state_proof/test_state_proof_for_get_requests.py` and `indy_node/test/state_proof/test_state_proof_for_missing_data.py` checking that we can query existing (and non-existing) Contexts from 1 Node only (using BLS multi-sig and state proofs).
We may have to skip the tests until Context is supported in Indy SDK.

Reply:
I added a test to `indy_node/test/state_proof/test_state_proof_for_get_requests.py` for `GET_CONTEXT` in [033c7e3](https://github.com/hyperledger/indy-node/pull/1430/commits/033c7e31cdd8f6592c4115c5d89d1763fe80dbb5)
As you will see, it uses `nodeSet` rather than `nodeSetWithOneNodeResponding`
It seems to pass all of the assertions, but if I use `nodeSetWithOneNodeResponding` I get a `PoolLedgerTimeout`
Any thoughts as to what the problem might be?

Reply:
As I mentioned above, this will not work until Context object is supported on SDK side (that is SDK has a logic for verification of BLS multi-signed state proofs for the Context object). Since SDK doesn't have Context object support yet, it always fallbacks to read from all Nodes, and reading from 1 Node raises `PoolLedgerTimeout`.